src/styles/blocks/page.scss

```diff
@@ -0,0 +1,5 @@
+html {
```

Comment:
It would be better to place this rule in main.scss, this is not a separate block